gubernator/main.py

```diff
@@ -91,6 +110,24 @@ def get(self):
         self.response.headers['Content-Type'] = 'text/plain'
         self.response.write('Warmup successful')
 
+
+class ConfigHandler(view_base.BaseHandler):
+    """Handles admin config of gubernator."""
+    def get(self):
+        self.render('config.html', {'hostname': hostname})
+
+    def post(self):
+        if users.is_current_user_admin():
```

Comment:
How does GAE handle this? How do I `POST` as the admin?

Reply:
It does an oauth login for your Google account. You use a browser.